<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-V59JQD1H" failures="6" tests="14" name="tests.Runner2" time="36.774" errors="1" timestamp="2023-11-16T19:45:14 IST" skipped="0">
  <testcase classname="tests.Runner2" name="runScenario" time="0.961">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at gluecode.StepDefs2.method1(StepDefs2.java:43)
at ✽.status code is "400" as per "blank" in json response body(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="1.557">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at gluecode.StepDefs4.method2(StepDefs4.java:40)
at ✽.status code is 204(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature4.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="1.694"/>
  <testcase classname="tests.Runner2" name="runScenario" time="5.286">
    <error message="Wrong line number" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: Wrong line number
at utilities.TextFilesUtility.getValuesOfLine(TextFilesUtility.java:32)
at gluecode.StepDefs4.method12(StepDefs4.java:50)
at ✽.submit request via DELETE to service by taking data from text file(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature4.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="1.092">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "200" as per "duplicate" and content type is "application/json"(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="4.643">
    <failure message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at gluecode.StepDefs3.method1(StepDefs3.java:77)
at ✽.validate that restful service with content type as "application/json"(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature3.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="1.696"/>
  <testcase classname="tests.Runner2" name="runScenario" time="0.984"/>
  <testcase classname="tests.Runner2" name="runScenario" time="1.013"/>
  <testcase classname="tests.Runner2" name="runScenario" time="1.818"/>
  <testcase classname="tests.Runner2" name="runScenario" time="6.682">
    <failure message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at gluecode.StepDefs5.method3(StepDefs5.java:112)
at ✽.validate response as per data in excel file(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature5.feature:18)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="tests.Runner2" name="runScenario" time="1.630"/>
  <testcase classname="tests.Runner2" name="runScenario" time="6.446"/>
  <testcase classname="tests.Runner2" name="runScenario" time="1.272">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at gluecode.StepDefs2.method3(StepDefs2.java:88)
at ✽.status code is "400" as per "wrong" and content type is "application/json"(file:///D:/Interview/API_Project_Using_Restassured-BDDcucumber-pico-testng/src/test/resources/features/feature2.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- tests.Runner2 -->
